.github/QUICK-START.md

```diff
@@ -0,0 +1,66 @@
+Install the components you want along with their dependencies. Here's an example:
+
+```shell
+yarn add -DW @spectrum-css/tokens @spectrum-css/typography @spectrum-css/page @spectrum-css/icon @spectrum-css/button
+```
+
+Spectrum CSS components utilize custom properties in order to change themes and scales. For these to apply, a couple of classes need to be added to the document’s `<html>` tag based on the [visual language](https://github.com/adobe/spectrum-css?tab=readme-ov-file#visual-language), [scale](https://github.com/adobe/spectrum-css?tab=readme-ov-file#scales), and [theme](https://github.com/adobe/spectrum-css?tab=readme-ov-file#themes-colorstops) you wish to use. For example, the following code snippet will display styling for the default Spectrum visual language using medium scale and light color theme:
+
+```html
+<html class="spectrum spectrum--medium spectrum--light">
+```
+
+Use the `index.css` files in your project to include component and global styles ([Spectrum and Express contexts](https://github.com/adobe/spectrum-css?tab=readme-ov-file#visual-language), [background theme/colorstop](https://github.com/adobe/spectrum-css?tab=readme-ov-file#themes-colorstops), [platform scaling](https://github.com/adobe/spectrum-css?tab=readme-ov-file#scales), etc.) for the component. If you don't need all of the global styles, peek at the docs for [including assets](https://github.com/adobe/spectrum-css?tab=readme-ov-file#including-assets)). Use this file by including the stylesheet (plus stylesheets for dependencies) in the `<head>` tag:
+
+```html
+<head>
+  <!-- Include global tokens depedency first -->
+  <link rel="stylesheet" href="node_modules/@spectrum-css/tokens/dist/index.css"/>
+
+  <!-- Include index.css for the components you're using -->
+  <link rel="stylesheet" href="node_modules/@spectrum-css/button/dist/index.css"/>
+</head>
+```
+
+Inside the `<body>` tag, add the markup for your component (Spectrum button in our example). The example also includes the CSS class names `spectrum-Button--fill` and `spectrum-Button--accent`, to use the accent variant:
+
+```html
+<button class="spectrum-Button spectrum-Button--fill spectrum-Button--accent spectrum-Button--sizeM">
+  <span class="spectrum-Button-label">Button</span>
+</button>
+```
+
+To put it all together, your final html file will look like this:
+
+```html
+<html class="spectrum spectrum--light spectrum--medium">
+  <head>
+    <link rel="stylesheet" href="node_modules/@spectrum-css/tokens/dist/index.css"/>
+    <link rel="stylesheet" href="node_modules/@spectrum-css/page/dist/index-vars.css">
+    <link rel="stylesheet" href="node_modules/@spectrum-css/button/dist/index-vars.css">
+  </head>
+  <body>
+    <button class="spectrum-Button spectrum-Button--fill spectrum-Button--accent spectrum-Button--sizeM">
+      <span class="spectrum-Button-label">Button</span>
+    </button>
+  </body>
+</html>
+```
+
+## Option 2: Include files from a CDN
```

Comment:
I'm not sure about including the CDN information here. Since unpkg and other CDN services are externally managed, we can't confirm their availability or up-time. If we had our own CDN, I'd say we could document it. The other thing is, we don't want anyone hosting these files from a CDN in a production environment.

Reply:
This is a great point. I added this section as a response to [some feedback](https://adobedesign.slack.com/archives/C5N154FEY/p1710946376240839?thread_ts=1710340701.269509&cid=C5N154FEY) I got from one of our users. But I'm not sure there's a way to document what they're asking for since, like you said, we don't have our own CDN.

Reply:
I tried something here, let me know what you think of this update: https://github.com/adobe/spectrum-css/pull/2598/files#diff-2c32729d6730631877f2e45f96395d8f4de2d78b83584dde2328396b6b84be27R55-R56

Reply:
Oh I love it!  That's perfect.